Dockerfile

```diff
@@ -75,6 +75,8 @@ RUN wget https://download.01.org/intel-sgx/sgx-linux/2.15/distro/ubuntu20.04-ser
   && rm $SGX_SDK_INSTALLER \
   && ls -l /opt/intel/
 
+ARG CTK_TAG
```

Comment:
Should we set some default value here in case someone wants to build this directly without the Makefile? Like just checkout the main branch?

Reply:
Yes, I agree. Fixed by setting it to master.